apps/teams-test-app/public/manifest.json

```diff
@@ -1,10 +1,52 @@
 {
-  "short_name": "Teams Test App",
-  "name": "Teams Test App",
-  "icons": [
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "version": "1.0.0",
+  "manifestVersion": "1.16",
+  "id": "a79a2374-ec2a-486e-a3dc-b913fdb1f365",
+  "packageName": "com.package.name",
+  "name": { "short": "M365 Test App", "full": "" },
+  "developer": {
+    "name": "M365",
+    "mpnId": "",
+    "websiteUrl": "https://<REPLACE WITH URL OF THE APP>",
+    "privacyUrl": "https://<REPLACE WITH URL OF THE APP>",
+    "termsOfUseUrl": "https://<REPLACE WITH URL OF THE APP>"
+  },
+  "description": {
+    "short": "App to test infrastructure",
+    "full": "App to test infrastructure"
+  },
+  "icons": { "outline": "outline.png", "color": "color.png" },
+  "accentColor": "#FFFFFF",
+  "configurableTabs": [
+    {
+      "configurationUrl": "https://<REPLACE WITH URL OF THE APP>",
+      "canUpdateConfiguration": true,
+      "scopes": ["groupChat", "team"],
+      "context": [
+        "channelTab",
+        "privateChatTab",
+        "meetingSidePanel",
+        "meetingStage",
+        "meetingDetailsTab",
+        "meetingChatTab"
+      ]
+    }
   ],
-  "start_url": "/",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
+  "staticTabs": [
+    {
+      "entityId": "test",
+      "name": "Test",
+      "contentUrl": "https://<REPLACE WITH URL OF THE APP>",
+      "websiteUrl": "https://<REPLACE WITH URL OF THE APP>",
+      "scopes": ["personal"]
+    },
+    { "entityId": "about", "scopes": ["personal"] }
+  ],
+  "validDomains": [],
+  "webApplicationInfo": {
+    "id": "34d8e67c-2758-40b5-836f-eb7defa875d3",
```

Comment:
Is this actually needed for anything? If you expect this app to also be usable for SSO (i.e. calls to getAuthToken), then the resource below needs to also be completed. Otherwise, I would recommend removing this whole section.